plugins/catalog-react/src/components/EntityOwnerPicker/EntityOwnerPicker.tsx

```diff
@@ -91,6 +101,28 @@ export const EntityOwnerPicker = () => {
     [backendEntities],
   );
 
+  const {
+    loading,
+    error,
+    value: ownerEntities,
+  } = useAsync(async () => {
+    const { items } = await catalogApi.getEntitiesByRefs({
+      entityRefs: availableOwners.map(ref =>
+        stringifyEntityRef(parseEntityRef(ref, { defaultKind: 'Group' })),
+      ),
+      fields: [
+        'kind',
+        'metadata.name',
```

Comment:
Forgot one comment: you want to have `metadata.namespace` here too to be able to properly make entity refs out of them.

Reply:
Good catch, will add.